### ISSUE TITLE
kendo-maskedtextbox cannot apply a mask when [includeLiterals]=true

### ISSUE THREAD

**Issue by anton-marchenko** [telerik/kendo-angular#3360]
Angular v9
Kendo "@progress/kendo-angular-inputs": "^6.5.0"

There is code:

```
@Component({
    selector: 'my-app',
    template: `
        <div class="example-config">
            <input id="ac" type="checkbox" [(ngModel)]="includeLiterals">
            <label for="ac">Include literals in the value</label>
        </div>
        <div class="example-wrapper">
            <kendo-maskedtextbox
              [includeLiterals]="includeLiterals"
              [(value)]="value"
              [mask]="mask">
            </kendo-maskedtextbox>
            <button (click)="onChange()">CLICK ME</button>
            <div>Component value: {{value}}</div>
            <div>Expected value: {{expectedValue}}</div>
  
        </div>
    `
})

export class AppComponent {
    public includeLiterals = true; // 1. mask does not apply when includeLiterals=true by default
    public value = '123';
    public expectedValue = '1-2-3';
    public mask = '0-0-0-0-0-0-0-0-0-0-0-0-0-0';

    onChange() {
        this.value += '8'; // 2. - mask does not apply here!
        this.expectedValue += '-8';
    }
}
```

When I try to change `value` **programmatically** ( fire `onChange()` function) the mask does not apply:
The mask applies and work correct when `includeLiterals ` reset by checkbox in user interface (NOT **programmatically**) or when `value ` changes by user interface (typing value by keyboard inside textbox)

There is [stackblitz example](https://stackblitz.com/edit/knd-mask-error?file=app/app.component.ts). Click button "CLICK ME" for reproduce;

**Expected behavior**
the mask applies programmatically by `onChange()` function when includeLiterals=true

**Browser**
 - Browser latest Chrome

**Comment by slavenai** [comment]
The includeLiterals property does not work as expected when it is initially set to true.

**Comment by Raisolution** [comment]
Fix available in `v8.0.9` of the Inputs package.
